content/writing-packages.md

```diff
@@ -19,11 +20,72 @@ With the release of version 1.3, Meteor has full support for npm. In the future,
 
 If you want to distribute and reuse code that you've written for a Meteor application, then you should consider publishing that code on npm if it's general enough to be consumed by a wider JavaScript audience. It's simple to [use npm packages from Meteor applications](using-packages.html#npm), and possible to [use npm packages from Atmosphere packages](#npm-dependencies), so even if your main audience is Meteor developers, npm might be the best choice.
 
-The practice of writing npm packages is [well documented](https://docs.npmjs.com/getting-started/creating-node-modules) and we won't cover it here.
-
 However, if your package depends on an Atmosphere package (which, in Meteor 1.3, includes the Meteor core packages), or needs to take advantage of Meteor's [build system](build-tool.html), then writing an Atmosphere package might be the best option.
 
-This article will cover some tips on how to do that.
+<h2 id="creating-npm">Creating an npm package</h2>
+
+```bash
+mkdir my-package
+cd my-package/
+meteor npm init
+```
+
+The last command creates a `package.json` file and prompts you for the package information. You may skip everything but `name`, `version`, and `entry point`. You can use the default `index.js` for `entry point`. This file is where you set your package's exports:
+
+```js
+// my-package/index.js
+exports.myPackageLog = function() {
+  console.log("logged from my-package");
+};
+```
+
+Now apps that include this package can do:
+
+```js
+import { myPackageLog } from 'my-package'
+
+myPackageLog(); // > "logged from my-package"
+```
+
+<h3 id="including-in-app">Including in your app</h3>
+
+When you are developing a new npm package for your app, there are a couple ways to include the package in your app:
+
+1. Place the package in your app's `node_modules/` directory, and add the package to source control. Do this when you want everything in a single repository.
+1. Place the package outside your app's directory in a separate repository and use `npm link`. Do this when you want to use the package in multiple apps.
```

Comment:
Can we link to the docs for `npm link`? Also I think it's important to mention this won't go out to others developing your app.